src/ngx_http_lua_directive.c

```diff
@@ -1572,7 +1572,7 @@ ngx_http_lua_conf_read_lua_token(ngx_conf_t *cf,
             }
 
             if (len) {
-                ngx_memcpy(b->start, b->pos, len);
+                ngx_memmove(b->start, b->pos, len);
```

Comment:
We should mention this fix in the commit log message.

Reply:
already added in the commit log.